src/components/data/DataTable.js

```diff
@@ -215,6 +213,11 @@ const DataTable = (props) => {
   const getAllEntities = async () => {
     const params = _.pickBy(_.trim, { pageSize: filteredCount, filterTerms: activeTextFilter, filterOperator });
     const queryResults = await Ajax(signal).Workspaces.workspace(namespace, name).paginatedEntitiesOfType(entityType, params);
+    Ajax().Metrics.captureEvent(Events.workspaceDataColumnTableSearch, {
+      ...extractWorkspaceDetails(workspace.workspace),
+      searchType: 'full-table-search',
+      providerName: dataProvider.providerName,
+    });
```

Comment:
I'm not getting this metric to fire - it's supposed to be when searching above the table (next to Advanced Search), right?  Also if it's full table search does that count as "data column table search"?

Reply:
Oh, yeah, this `getAllEntities` function is called when clearing a column and selecting all rows.

Reply:
I think https://github.com/DataBiosphere/terra-ui/blob/49469212a82b09b13599aace22afaa82eeca71cd/src/components/data/DataTable.js#L388 might be the place for this event.

Reply:
So I'd add the captureEvent call to onChange? 

(we might want to look at UI ramp up tickets cause this ticket does explicitly call out 'filterTerms' as the place to put the event)

Reply:
That does seem to work :)

Reply:
Yes.

It looks like the `activeTextFilter` state variable gets translated to the `filterTerms` query parameter in EntityServiceDataTableProvider.